torch/csrc/jit/passes/quantization.cpp

```diff
@@ -64,6 +64,18 @@ struct PatternsAndModules {
   Module packed_params_module;
 };
 
+std::vector<std::string> _single_input_general_call_funcs = {
```

Comment:
Can you add a comment to explain what these mean? Is the first list functions that do dispatch for qtensor under the hood, so that you can propagate dq?

Reply:
will do